source/app/service-providers/fsal/fsal-directory.ts

```diff
@@ -399,8 +401,14 @@ export async function renameChild (
   newName: string,
   parser: (file: MDFileDescriptor, content: string) => void,
   sorter: (arr: AnyDescriptor[], sortingType?: string) => AnyDescriptor[],
-  cache: FSALCache
+  cache: FSALCache,
+  allowOverwrite: boolean = false
```

Comment:
I would suggest renaming this parameter to `forceOverwrite`. The reason is that, somehow (this may just be my personal preference, so feel free to comment on this suggestion by me), `allowOverwrite` sounds to me as if there was user choice involved, whereas `forceOverwrite` has a more programmatic sound to it.

Since we now do have the proper checks in the file renaming command, I don't think we ever need to ask the user to set this parameter. Instead, it makes sense to include this parameter purely for potential future use if we at some point allow Zettlr to programmatically rename files. Since it works properly in this implementation, I'd suggest we definitely keep it for future use, and if the parameter is never used, then, well, so be it.

Reply:
I think that this change makes sense, change should be part of the pull request now.

Reply:
Wonderful, thank you!